docs/quadratic_programming.rst

```diff
@@ -0,0 +1,316 @@
+Quadratic programming
+=====================
+
```

Comment:
This section is concerned with minimizing quadratic functions subject to equality and/or inequality constraints, also known as quadratic programming.